runtime/doc/news.txt

```diff
@@ -435,10 +435,8 @@ The following changes to existing APIs or features add new behavior.
   • Enabled treesitter highlighting for Lua files.
 
 • The `workspace/didChangeWatchedFiles` LSP client capability is now enabled
-  by default.
-  • On Mac or Windows, `libuv.fs_watch` is used as the backend.
-  • On Linux, `fswatch` (recommended) is used as the backend if available,
-    otherwise `libuv.fs_event` is used on each subdirectory.
+  by default on Mac and Windows. Disabled on Linux since there currently isn't
```

Comment:
I'd phrase this a bit different and mention that it doesn't scale well with large amount of directories. 
For small to moderate sized projects it's okay. I use it all the time with no noticeable performance impact

Reply:
Can do, but I wouldn't consider the testcase in https://github.com/neovim/neovim/issues/23291#issuecomment-1929134544 to be large, and I don't think pyright requesting to watch the entire virtual env to be unreasonable.